buildroot/board/meraki/ms220/overlay/bin/configd

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env sh
+#
+# daemon to watch /etc/switch.json for changes
+#
+
+# source of truth
+CONFIG=/etc/switch.json
+# cache file to diff changes
+CACHE=/etc/.switch.cache.json
```

Comment:
`/etc/` is an overlay filesystem on NOR flash. I think it would be better to locate the cache file on ephemeral storage, like in `/tmp/` or `/dev/shm` as it does not require persistence.

Reply:
Makes sense -- moved to `/tmp`. I was originally using `mktemp` but realized that would bust the cache everytime the daemon is restarted. Probably less on a problem outside of dev though and moving it to `/tmp` would only reset on every reboot..